### PR TITLE
Use Scholar's "What are you uploading?" page

### LINKED ISSUE #249: Use Scholar's "What are you uploading?" page
### Descriptive summary

Instead of displaying a pop-up modal of available work types when a user creates a work, show a page like Scholar's "What are you uploading?" page.  See https://scholar.uc.edu/classify_concerns/new

If a user tries to access that page when they are not logged in, they should be sent to the /login page.

The commit that implemented this in Scholar can be found here:
https://github.com/uclibs/scholar_uc/commit/bb5506f279cc05341191d55ff905c627e124da52

### PR DESCRIPTION
**Pull request by changy66**
Fixes #249 

Added view file for uploading page
Added classifyConcern controller and models
Changed creating work type feature test

Deleted 'human_readable_short_description' in models and used locals instead.

Changed the work type in local file from using space to using first letter capital. For example, 'student_work' changed to 'StudentWork'. So the variable 'klass' can be used in 'new.html' to represent work type.

### CODE REVIEW

**Comment by hortongn** [comment]
@changy66 This PR has merge conflicts.  Please rebase.  Thanks!

**Comment by hortongn** [comment]
@changy66 when you are all done with this branch, please squash your commits.

File: app/models/article.rb [hunk 1]
```diff
@@ -6,6 +6,8 @@ class Article < ActiveFedora::Base
   include ::Hyrax::WorkBehavior
 
   self.indexer = ArticleIndexer
+  self.human_readable_short_description = 'Published or unpublished articles'
```
Comment (hortongn) [inline app/models/article.rb]: @changy66 Another change: The description for each work type should come from config/locales.  For example, the description for articles should come from `en.hyrax.select_type.description`.  See https://github.com/uclibs/ucrate/blob/develop/config/locales/article.en.yml#L8

**Comment by changy66** [comment]
I see. Thanks, Glen

On Jul 9, 2018, at 4:47 PM, Glen Horton <notifications@github.com<mailto:notifications@github.com>> wrote:


@hortongn commented on this pull request.

________________________________

In app/models/article.rb<https://github.com/uclibs/ucrate/pull/269#discussion_r201141778>:

> @@ -6,6 +6,8 @@ class Article < ActiveFedora::Base
   include ::Hyrax::WorkBehavior

   self.indexer = ArticleIndexer
+  self.human_readable_short_description = 'Published or unpublished articles'


@changy66<https://github.com/changy66> Another change: The description for each work type should come from config/locales. For example, the description for articles should come from en.hyrax.select_type.description. See https://github.com/uclibs/ucrate/blob/develop/config/locales/article.en.yml#L8

**Pull request merged**